src/components/MediaPlayer/MainControl/PlayerButtons/PlayerButtons.js

```diff
@@ -25,6 +25,8 @@ export class PlayerButtonsComponent {
   finishPlay() {
     this.isPlayNow = false;
     this.playButton.children[0].innerText = "play_arrow";
+    this.props.onPlayerChangeState(false);
```

Comment:
Pause button in tables and in a player wasn't synchronized at the moment song was finished. Thank's to this line they are synchronized right now.